Assets/Scripts/WorldProperties.cs

```diff
@@ -91,6 +93,111 @@ void Start()
             clipShader = GameObject.FindWithTag("Ground").GetComponent<Renderer>().material.shader;
         }
 
+        /// <summary>
+        /// Cycle through the connected drones
+        /// </summary>
+        public static void SelectNextDrone()
+        {
+            Debug.Log("Selection next drone");
+
+            if(selectedDrone != null)
+            {
+                dronesQueue.Enqueue(selectedDrone);
+            }
+
+            if (dronesQueue.Count > 0)
+            {
+                Drone nextDrone = dronesQueue.Dequeue();
+                nextDrone.gameObjectPointer.GetComponent<DroneProperties>().SelectDrone();
+            }
+            else
+            {
+                Debug.Log("No drones connected");
+            }
+
+        }
+
+        /// <summary> 
+        /// Get nextDroneID 
+        /// </summary>
+        public static char GetNextDroneID()
```

Comment:
Since it's only called once, GetNextDroneID should be combined with increment so it increments and then returns the next ID. Making it this way would be one less "gotcha"!